frontend/app/services/subscription-service.server.ts

```diff
@@ -29,19 +29,47 @@ function createSubscriptionService() {
     const auditService = getAuditService();
     const instrumentationService = getInstrumentationService();
     auditService.audit('alert-subscription.get', { userId });
-    //TODO, for a future PR, use HATEOAS links to get the /subscriptions endpoint
-    const url = new URL(`${CDCP_API_BASE_URI}/api/v1/users/${userId}/subscriptions`);
-
+    const url = new URL(`${CDCP_API_BASE_URI}/api/v1/users/${userId}`);
     const response = await fetch(url, {
       method: 'GET',
       headers: {
         'Content-Type': 'application/json',
       },
     });
 
+    const userSchema = z.object({
+      id: z.string(),
+      email: z.string(),
+      emailVerified: z.boolean(),
+      userAttributes: z.array(
+        z.object({
+          name: z.string(),
+          value: z.string(),
+        }),
+      ),
+      _links: z.object({
+        self: z.object({
+          href: z.string(),
+        }),
+        subscriptions: z.object({
+          href: z.string(),
+        }),
+      }),
+    });
+    instrumentationService.countHttpStatus('http.client.cdcp-api.user.gets', response.status);
```

Comment:
Since we're getting the `users` resource:

```suggestion
    instrumentationService.countHttpStatus('http.client.cdcp-api.users.gets', response.status);
```